src/tiled/changelayer.cpp

```diff
@@ -108,6 +108,7 @@ void SetLayerLocked::swap()
     emit mDocument->changed(LayerChangeEvent(mLayer, LayerChangeEvent::LockedProperty));
 }
 
+
```

Comment:
Ha, thank you for caring about consistency!

Reply:
=)